app/BWT.py

```diff
@@ -114,6 +114,19 @@ def __init__(self, aligner, include_wildcard, include_baits, read_one, read_two,
 
 		self.output_tab = os.path.join(self.working_directory, "{}.temp.txt".format(self.output_file))
 		self.output_tab_sequences = os.path.join(self.working_directory, "{}.seqs.temp.txt".format(self.output_file))
+		# Parse tab-delimited file into dictionary for mapped reads
+		self.alignments = {}
+		with open(self.output_tab_sequences, 'r') as csvfile:
```

Comment:
I moved the code for parsing the tab file to a dictionary to the BWT initialization.
However, at this time, the file ('{}.seqs.temp.txt'.format(self.output_file)) has not been generated yet.
This issue is fixed in the next commit (17dc039), which introduces a function called 'BWT.preload_alignments'. 
Once the '.seqs.temp.txt' file is generated, it will be parsed.

Reply:
Ok, @hunglin59638 got it. Thanks.